docs/index.md

```diff
@@ -1451,6 +1517,13 @@ $ REPORTER=nyan npm test
 
 In addition to chatting with us on [Gitter], for additional information such as using spies, mocking, and shared behaviours be sure to check out the [Mocha Wiki](https://github.com/mochajs/mocha/wiki) on GitHub. For discussions join the [Google Group](https://groups.google.com/group/mochajs). For a running example of Mocha, view [example/tests.html](example/tests.html). For the JavaScript API, view the [API documentation](api/) or the [source](https://github.com/mochajs/mocha/blob/master/lib/mocha.js#L51).
 
+[//]: # (Cross reference section)
+
+[Growl]: http://growl.info/
+[Growl-install]: https://github.com/mochajs/mocha/wiki/Growl-Notifications
+[Mocha-gitter]: https://gitter.im/mochajs/mocha
+[Mocha-website]: https://mochajs.org/
+
```

Comment:
@boneskull,
I have a different PR out for updating this documentation file to use cross references throughout. I had added a couple in this PR, but was caught off-guard when asked to fix merge conflict. Where did the `AUTO-GENERATED-CONTENT` part below come from? I merged compatibly retaining both (for the moment).

Reply:
`AUTO-GENERATED-CONTENT` is [markdown-magic](https://npm.im/markdown-magic) stuff.  I was showing it's easy to use to automatically pull data out of `package.json`, knowing that we'd want to do more of it... give it a shot; you can insert the link to the mochajs homepage in the same manner as the gitter one by modifying the template (or adding a new `<!-- AUTO-GENERATED:START --> .. <!-- AUTO-GENERATED:END -->` block)

Reply:
run `npm start docs.preprocess` to apply our set modifications.  see `scripts/markdown-magic.config.js`